.github/workflows/ci.yml

```diff
@@ -0,0 +1,103 @@
+            - name: Download Stylua
+              run: |
+                  mkdir /tmp/stylua && cd $_
+                  curl -fsSL -o stylua.zip "https://github.com/JohnnyMorganz/StyLua/releases/download/v0.17.0/stylua-linux.zip"
+                  unzip -d /usr/local/bin stylua.zip
+            - name: Download Shellharden
+              run: |
+                  mkdir /tmp/shellharden && cd $_
+                  curl -fsSL -o shellharden.tar.gz https://github.com/alsuren/cargo-quickinstall/releases/download/shellharden-4.2.0-x86_64-unknown-linux-gnu/shellharden-4.2.0-x86_64-unknown-linux-gnu.tar.gz
+                  tar -xvf shellharden.tar.gz
+                  mv shellharden /usr/local/bin/
+            - name: Run cbfmt check
+              # Lua examples in README.md doesn't conform to Stylua rules, on purpose.
+              run: find . -name '*.md' -not -path './dependencies/*' -not -path './README.md' | xargs cbfmt --check
+    selene:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: Run Selene check
+              uses: NTBBloodbath/selene-action@v1.0.0
+              with:
+                  # token is needed because the action allegedly downloads binary from github releases
+                  token: ${{ secrets.GITHUB_TOKEN }}
+                  args: lua/ tests/ scripts/
+                  version: 0.24.0
+    stylua:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: Run Stylua check
+              uses: JohnnyMorganz/stylua-action@v1.1.1
+              with:
+                  # token is needed because the action allegedly downloads binary from github releases
+                  token: ${{ secrets.GITHUB_TOKEN }}
+                  # CLI arguments
+                  args: --check .
+                  version: 0.17.0
+    tests:
+        strategy:
+            fail-fast: false
+            matrix:
+                nvim_version:
+                    - v0.7.0
+                    - v0.7.2
+                    - v0.8.0
+                    - v0.8.1
+                    - v0.8.2
+                    - v0.8.3
+                    - v0.9.0
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: rhysd/action-setup-vim@v1
+              with:
+                  neovim: true
+                  version: ${{ matrix.nvim_version }}
+            - name: Run tests
+              run: |
+                set -e
+                make test
+                nvim -u NONE -E -R --headless +'helptags doc' +q
+    release:
+      name: release
+      if: ${{ github.ref == 'refs/heads/main' }}
+      needs:
+        - cbfmt
+        - selene
+        - stylua
+        - tests
+      runs-on: ubuntu-latest
+      steps:
+        - uses: google-github-actions/release-please-action@v3
+          id: release
+          with:
+            token: ${{ secrets.RELEASE_TOKEN }}
```

Comment:
Can we use the default secret (`GITHUB_TOKEN`) instead (i.e. omit this entirely)?

Reply:
yes, sorry. bad copy paste